source/admin_manual/cassandra/cassandra.rst

```diff
@@ -38,3 +38,24 @@ In the Dovecot Pro log file, if you start seeing ``Object exists in dict, but no
 Resurrected deleted data will have to be manually deleted.
 
 To prevent Cassandra data resurrection, you must regularly run ``nodetool repair`` within ``gc_grace_seconds`` via cron for the entire cluster.
+
+Reducing Tombstones
+===================
+
+Dovecot attempts to prevent creating too many tombstones within the same
+cluster key. Sometimes it may not have worked properly though, and Cassandra
+queries start failing (timing out) towards a specific cluster key due to too
+many tombstones. This can be repaired by getting rid of the tombstones:
+
+ * Run Cassandra repair to make sure all tombstones are replicated
```

Comment:
I'm not that sure of Cassandra administration either. I think either command line tool or maybe there is some other kind of a tool also. Either way, since I'm not 100% sure myself I'll just leave it vague so people running Cassandra will know what to do. :)